plugins/mmstd/include/mmstd/flags/FlagStorage.h

```diff
@@ -122,7 +122,10 @@ class FlagStorage : public core::Module {
     core::param::ParamSlot serializedFlags;
 
     std::shared_ptr<FlagCollection_CPU> theCPUData;
-    uint32_t version = 0;
+
+    // set initial version (in first frame, with temp data) != 0 so call data comes out as "has Update"
+    // and eventually triggers a reload of actual user-provided data via validateFlagCount()
+    uint32_t version = 1;
```

Comment:
Is there a specific module that requires this or a general fix? Should validateFlagCount() maybe be called independent from hasUpdate? E.g. when changing data (size) this will also not be reflected by an update inside the flag storage and instead connected modules need to inform the flag storage about the change in size.

Reply:
I think initially i fixed this for the SphereRenderer+PCP use case, where time dependent particle data did not get updated via the FlagStorage.